rudder-server-relay/debian/control

```diff
@@ -8,8 +8,8 @@ Homepage: http://www.rudder-project.org
 
 Package: rudder-server-relay
 Architecture: any
-Depends: ${shlibs:Depends}, ${misc:Depends}, python, rudder-agent, apache2, apache2-utils, rsyslog, openssl, libapache2-mod-wsgi
+Depends: ${shlibs:Depends}, ${misc:Depends}, python, rudder-agent (>= ${binary:Version}), apache2, apache2-utils, rsyslog, openssl, libapache2-mod-wsgi
```

Comment:
I don't think we should depend on a specific agent version except for a good reason